Ui/Component/Listing/Columns/CrossBorderClassActions.php

```diff
@@ -57,6 +65,16 @@ public function prepareDataSource(array $dataSource)
     {
         if (isset($dataSource['data']['items'])) {
             foreach ($dataSource['data']['items'] as &$item) {
+
+                /**
+                 * overriding cross border type id int with the label
+                 */
+                if(isset($item['cross_border_type_id'])) {
+                    $item['cross_border_type_id'] = $this->fetchCrossBorderTypeValue(
```

Comment:
This method can throw an exception, which you should handle here.

Reply:
instead of adding exception handling, I removed the dependency in the fetchCrossBorderTypeValue method, which is what caused the possibility for an exception.

Reply:
@LordZardeck per our conversation I've added handling for the localized exception, changed the variable type_id to camelcase, and updated the exception to use a constant that is set as "unknown".